llvm/test/Transforms/LoopVectorize/AArch64/sve2-histcnt.ll

```diff
@@ -753,6 +753,29 @@ for.exit:
   ret void
 }
 
+; The histogram operation generates vectors. This example used to crash
+; due to a missing entry in a switch statement.
+define void @histogram_generates_vectors_crash(ptr %data_array) {
+entry:
+  br label %for.body
+
+for.body:
+  %iv = phi i64 [ 0, %entry ], [ %iv.next, %for.body ]
+  %gep.indices = getelementptr [1048576 x i32], ptr null, i64 %iv
```

Comment:
Not sure what to do about this immediate UB, but this is the best test I can supply.

Reply:
Hmm, surprised this would be needed.

Reply:
Thanks!